src/commons/HTMLRenderer.js

```diff
@@ -182,10 +182,8 @@ export default class HTMLRenderer extends React.Component {
               a: ({ href }, children, _, { onLinkPress, key }) => {
                 if (!href.includes('http')) return null;
                 return (
-                  <TouchableOpacity
+                  <Text

```

Comment:
I had to use TouchableOpacity for IOS so let's add an if and check if it's ios render the TouchableOpacity and on android let's render only the Text